source/Engine.cpp

```diff
@@ -1461,10 +1468,23 @@ void Engine::CalculateStep()
 
 	// Keep track of the flagship to see if it jumps or enters a wormhole this turn.
 	const Ship *flagship = player.Flagship();
+	bool flagshipWasUntargetable = (flagship && !flagship->IsTargetable());
 	bool wasHyperspacing = (flagship && flagship->IsEnteringHyperspace());
-	// Move all the ships.
+	// First, move the player's flagship.
+	MoveShip(player.FlagshipPtr());
+	// Then, move all the rest of the ships.
```

Comment:
```suggestion
	// Then, move all the other ships.
```
sounded strange to me, but then on the other hand my english is meh

Reply:
I'd say either "move the rest of the ships" or "move the other ships." Saying "move all the rest of the ships" does sound strange to me, but so does "all the other ships"; "all" and "the rest" or "the other" are redundant with one another.
```suggestion
	// Then, move the other ships.
```